sql/1.107/contributor_activity.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+CREATE TABLE contributor_activity (
+    id serial NOT NULL PRIMARY KEY,
+    github_id text NOT NULL UNIQUE,
```

Comment:
that's a good point! So then I'll remove `Id` and set `github_id` to the Primary Key?